src/backend/utils/email-sender.js

```diff
@@ -15,58 +15,87 @@ const log = logger.child({ module: 'email-sender' });
+    to: receipients, // People to send to
+    subject: subjectMessage, // Subject Line
+    html:
+      '<body>' +
+      '<table border="1" cellpadding="0" cellspacing="0" width="100%">' +
+      '<td align="center" bgcolor="#70bbd9" style="padding: 40px 0 30px 0;">' +
+      '<h1 style="font-size:40px"><b>Seneca-CDOT Telescope</b></h1>' +
+      '<a href="https://github.com/Seneca-CDOT/telescope" style="font-size:30px">Github</a>' +
+      '</td>' +
+      '<td align="center" bgcolor="#db3d3d" style="padding: 40px 0 30px 0;">' +
+      '<h1 style="font-size:40px"><b>⚠️ERROR</b></h1>' +
+      '</td>' +
+      '<tr><td style="padding: 40px 30px 40px 30px;" colspan="2">' +
+      '<table cellpadding="0" cellspacing="0" width="100%" style="table-layout: fixed;">' +
+      '<tr><td width="15%"><b>Error Message:</b></td>' +
+      `<td style="word-wrap:break-word" width="85%">${message}</td></tr>` +
+      '</table></td>' +
+      '</table>' +
+      '</body>', // Message Body
+  };
 
-    // Email Content
-    const mailOptions = {
-      from: process.env.NODEMAILER_USERNAME, // Email Name
-      to: receipiants, // People to send to
-      subject: subjectMessage, // Subject Line
-      html:
-        '<body>' +
-        '<table border="1" cellpadding="0" cellspacing="0" width="100%">' +
-        '<td align="center" bgcolor="#70bbd9" style="padding: 40px 0 30px 0;">' +
-        '<h1 style="font-size:40px"><b>Seneca-CDOT Telescope</b></h1>' +
-        '<a href="https://github.com/Seneca-CDOT/telescope" style="font-size:30px">Github</a>' +
-        '</td>' +
-        '<td align="center" bgcolor="#db3d3d" style="padding: 40px 0 30px 0;">' +
-        '<h1 style="font-size:40px"><b>⚠️ERROR</b></h1>' +
-        '</td>' +
-        '<tr><td style="padding: 40px 30px 40px 30px;" colspan="2">' +
-        '<table cellpadding="0" cellspacing="0" width="100%" style="table-layout: fixed;">' +
-        '<tr><td width="15%"><b>Error Message:</b></td>' +
-        `<td style="word-wrap:break-word" width="85%">${message}</td></tr>` +
-        '</table></td>' +
-        '</table>' +
-        '</body>', // Message Body
-    };
+  return mailOptions;
+};
+
+// Verifies if the transporter passed was created successfully
+exports.verifyTransporter = function(transporter) {
+  // Verify connection configuration
+  transporter.verify(err => {
+    // If error then print to console
+    if (err) {
+      log.error(err);
+      return false;
+    }
+    // else print a ready message
+    log.info('Server is running properly');
+    return true;
+  });
+};
+
+// Sends a message using the passed in parameters
+exports.sendMessage = async function(receipiants, subjectMessage, message) {
+  return new Promise((resolve, reject) => {
+    const transporter = this.createTransporter(
+      process.env.NODEMAILER_SERVER,
+      2222,
+      false,
+      process.env.NODEMAILER_USERNAME,
+      process.env.NODEMAILER_PASSWORD
```

Comment:
I was going to create an issue for this, but its a one liner. require('../lib/config'); needs to be included at the top to access the environment variables. I think its fine to include it in this PR since your restructuring the entire thing

Reply:
It's been added. Do you mind approving the changes then?

Reply:
It looks good to me, did you produce the coverage report? Does this cover all the functions for the coverage reports for the email-sender.js file?